slick-codegen/src/main/scala/slick/codegen/AbstractSourceCodeGenerator.scala

```diff
@@ -57,14 +57,39 @@ abstract class AbstractSourceCodeGenerator(model: m.Model)
       else compoundValue(types)
     }
 
-    def compoundValue(values: Seq[String]): String = {
+    val maxTupleSize = 22
+    //build and nest tuples if > 22 elements
+    def nestedTuple(values: Seq[String] , prefix: String): String = {
```

Comment:
I'd prefer if you'd stick to the whitespace-style we use almost everywhere. e.g. no space before comma